src/marks/mark.py

```diff
@@ -21,6 +21,8 @@
     "red": "red",
     "pink": "pink",
     "yellow": "yellow",
+    "-userColor1": "userColor1",
+    "-userColor2": "userColor2",
```

Comment:
Should we give these real spoken forms? Eg "navy" and whatever the closest match is for the default you've used for color 2? Also, I'd be inclined not to include the `-`, because they will only appear in the csv if the user has actually turned them on

Reply:
I'm unsure. I have really not found a good second color so whatever we name the second color is probably not going to be what the user actually ends up using. 

We can remove the minus.

Reply:
I'm just a bit hesitant to use camel case in the spoken form column.  Why not just call them "navy" and "salmon"?

Reply:
We can do that if you prefer it. I can see if I can find a better second color and then update the names.

Reply:
The second color looks close enough to salmon to me. I would be inclined to just change the spoken form to "salmon" and ship it.  But if you wanna keep tweaking the colors go for it

Reply:
I think it's peach or something like that. I will take another crack at it before deciding.

Reply:
I couldn't find a better color that didn't conflict with the others visually. But the colors are now renamed to navy and apricot.